.travis.yml

```diff
@@ -28,6 +28,7 @@ env:
     - PACKAGE=navi-directory
     - PACKAGE=navi-notifications
     - PACKAGE=navi-reports
+    - PACKAGE=navi-search
```

Comment:
Can we not publish search until it is ready?

Reply:
Keep this, but make it private in `package.json`

Reply:
will just marking it as private in the package json work or is there another option in the travis.yml that needs to be updated?